privacy/scrubber.go

```diff
@@ -260,6 +262,12 @@ func scrubIPV6Lowest16Bits(ip string) string {
 	return ip
 }
 
+func anonymizeIpv6(ip string) string {
+	ipv6Mask := net.CIDRMask(56, 128)
+	ipMasked := net.ParseIP(ip).Mask(ipv6Mask)
+	return ipMasked.String()
+}
```

Comment:
This function can be reused in other scrubbing strategies like 
```go
      // ScrubStrategyIPV6Lowest16 zeroes out the last 16 bits of an IPV6 address.
	ScrubStrategyIPV6Lowest16

	// ScrubStrategyIPV6Lowest32 zeroes out the last 32 bits of an IPV6 address.
	ScrubStrategyIPV6Lowest32
```
by passing number of bits that should remain un-anonymous to the first parameter here:`net.CIDRMask(56, 128)`

Reply:
Please change the name to `scrubIPV6` to match PBS-Go naming. This functionality should also replace `ScrubStrategyIPV6Lowest16` and `ScrubStrategyIPV6Lowest32` instead of exist beside it.